soc/arm/st_stm32/stm32f0/dts_fixup.h

```diff
@@ -60,6 +60,10 @@
 #define CONFIG_USB_NUM_BIDIR_ENDPOINTS		ST_STM32_USB_40005C00_NUM_BIDIR_ENDPOINTS
 #define CONFIG_USB_RAM_SIZE			ST_STM32_USB_40005C00_RAM_SIZE
 
+#ifdef ST_STM32_USB_40005C00_ENABLE_PIN_REMAP
```

Comment:
Why do you need the ifdef here?

Reply:
#ifdefs don't do chain resolution right?

e. g.:

```
#define SOMETHING SOMETHINGELSE

#if defined(SOMETHING)
#error "Wat?"
#endif
```

Fails to build.

Reply:
@galak , are you good?